sdk/eventhubs/ci.yml

```diff
@@ -20,18 +20,18 @@ pr:
       - hotfix/*
   paths:
     include:
-      - sdk/eventhubs/azure-messaging-eventhubs
+      - sdk/eventhubs
       - eng/common/testproxy
 
 stages:
   - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
     parameters:
-      ServiceDirectory: eventhubs/azure-messaging-eventhubs
+      ServiceDirectory: eventhubs
       CtestRegex: "azure-messaging-eventhubs.*"
       LiveTestCtestRegex: "azure-messaging-eventhubs.*"
       LiveTestTimeoutInMinutes: 120
-      LineCoverageTarget: 81
-      BranchCoverageTarget: 42
+      LineCoverageTarget: 22
```

Comment:
We better boost up this coverage soon, ideally before the release. `22` is pretty low.

Reply:
Not gonna happen without substantial engineering effort. There is no mechanism for recording AMQP traffic.

Reply:
Can we write these tests programmatically?